jruby-gradle-base-plugin/src/main/groovy/com/github/jrubygradle/JRubyPlugin.groovy

```diff
@@ -41,30 +41,28 @@ class JRubyPlugin implements Plugin<Project> {
             JRubyExec.updateJRubyDependencies(project)
         }
 
-        project.task('jrubyPrepareGems', type: JRubyPrepareGems) {
-            group TASK_GROUP_NAME
-            description 'Prepare the gems from the `gem` dependencies, extracts into jruby.installGemDir'
-            gems project.configurations.gems
-            outputDir project.jruby.gemInstallDir
-        }
+        Task jrpg = project.tasks.create 'jrubyPrepareGems'
```

Comment:
This is unnecessary FWIW, you can still use the task creation closure for generic tasks like this, e.g.

```groovy
Task jrpg = project.tasks.create('jrubyPrepareGems') {
  dependsOn 'jrubyPrepare'
  // etc
}
```

This doesn't need to change for this PR, just wanted to bring it up